src/db.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize({
+  dialect: 'postgres',
+  host: process.env.POSTGRES_HOST,
+  username: process.env.POSTGRES_USERNAME,
+  password: process.env.POSTGRES_PASSWORD,
+  database: process.env.POSTGRES_DATABASE,
+});
```

Comment:
Consider adding error handling for the database connection. Without it, if the connection fails due to incorrect credentials or network issues, the application might crash without a clear error message.